src/main/java/org/carlspring/strongbox/janusgraph/graph/gremlin/GremlinConfig.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.janusgraph.graph.gremlin;
+
+import java.io.InputStream;
+
+import org.apache.cassandra.service.CassandraDaemon;
+import org.apache.tinkerpop.gremlin.server.GremlinServer;
+import org.apache.tinkerpop.gremlin.server.Settings;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Configuration
+@ComponentScan
+public class GremlinConfig
+{
+
+    @Bean(destroyMethod = "stop")
+    @ConditionalOnMissingBean
+    GremlinServer gremlinServer(
+            CassandraDaemon cassandra,
+            @Value("classpath:conf/gremlin-server/gremlin-server.yaml") Resource gremlinServerConf,
+            @Value("classpath:conf/gremlin-server/janusgraph-cql-es-server.properties") Resource janusGraphBackendProperties)
```

Comment:
we better to expose configuration settings that we need with `application,yaml`, you can have a look at original `JanusGraphConfig` as an example.